core/runtime/impl/wasm_memory_impl.cpp

```diff
@@ -7,22 +7,25 @@
 
 namespace kagome::runtime {
 
-  WasmMemoryImpl::WasmMemoryImpl() {
-    offset_ = 0;
-    allocate(1);  // We should allocate very first byte to prohibit allocating
-                  // memory at 0 in future, as returning 0 from allocate method
-                  // means that wasm memory was exhausted
-  }
+  WasmMemoryImpl::WasmMemoryImpl()
+      : offset_{1}  // We should allocate very first byte to prohibit allocating
+                    // memory at 0 in future, as returning 0 from allocate
+                    // method means that wasm memory was exhausted
```

Comment:
where is `allocate`?

Reply:
It is not needed. Like the one who wrote this code says, it was just a hotfix, which led to the warning. So, just starting to allocate the memory from 1st and not 0th byte is enough